net/gatling/Makefile

```diff
@@ -32,9 +32,19 @@ define Package/gatling/description
   Gatling is particularly good in situations with very high load.
 endef
 
+define Package/conffiles
```

Comment:
```suggestion
define Package/gatling/conffiles
```

Reply:
Added in the new version of the commit. Thank you for pointing out.